app/models/user.rb

```diff
@@ -397,6 +399,10 @@ def country
     Country.find_by_iso2(country_iso2)
   end
 
+  def newcomer?
```

Comment:
Just curious, are you using this method anywhere except maybe in tests?

Reply:
Using it in tests, but also in `registration_checker:196`:
```ruby
          if !target_user.newcomer? && competition.newcomer_month_reserved_spots > 0
```

Reply:
Then that reference should probably be `if !target_user.newcomer_month_eligible?`, or not??!